README.md

```diff
@@ -1,56 +1,52 @@
-# O3DE-Extras
+# o3de-extras
 
-Extras add-on repo for O3DE.
+As the name implies, o3de-extras repo is for "extra" O3DE objects that are considered "canonical" to O3DE but not "core" to the engine.
+i.e. The core engine will build and run without the extras, however the o3de-extras contain extra projects, gems, etc. in a standard O3DE compound repo format.
+
+## Testing
+All O3DE extras gems are tested with the engine and follow the same coding standards and thus should be thought of as part of the engine.
+When testing is performed on the core engine, any gem enabled for testing in the o3de-extras .automatedtesting.json will have its tests run.
```

Comment:
We probably want to put something about the .automatedtesting file here and in the installation steps. Some of the keys are obvious, but it would be user friendly to describe them

Reply:
done